server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -618,6 +615,50 @@ export function configureWebSocketServices(
 				);
 			}
 
+			// Set up listener to forward signal to clients in the collaboration session when the broadcast-signal endpoint is called
+			collaborationSessionEventEmitter?.on(
+				"broadcastSignal",
+				// eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
Regarding this eslint rule, is there a way to implement the change without having to ignore the rule? I am not particularly sure if this was needed due to a missing `async` annotation in line 622 below or if we needed an await somewhere. But I know @znewton did a lot of work recently to clean up incorrectly configured promises that were causing a lot of unhandled promise rejections/uncaught exceptions and I am wondering if this is one of those cases we had to fix. So, it would be good to fix it right away instead of suppressing the eslint rule, if possible